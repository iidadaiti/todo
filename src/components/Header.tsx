import React, { ComponentProps, FC } from 'react'
import { mergeClassNames } from '../utils/classNames'
import styles from './Header.module.css'
import { Heading } from './Heading'

export const Header: FC<Omit<ComponentProps<'div'>, 'children'>> = ({
  className,
  ...props
}) => (
  <div className={mergeClassNames(styles.container, className)} {...props}>
    <Heading>Todo</Heading>
  </div>
)
