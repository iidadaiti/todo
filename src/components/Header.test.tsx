import { render } from '@testing-library/react'
import React from 'react'
import { Header } from './Header'

test('Header snapshot', () => {
  const result = render(<Header></Header>)
  expect(result.asFragment()).toMatchSnapshot()
})
