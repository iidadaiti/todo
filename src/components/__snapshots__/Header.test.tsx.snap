// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h1
      class="spectrum-Heading spectrum-Heading--sizeXXL"
    >
      Todo
    </h1>
  </div>
</DocumentFragment>
`;
